### DISCUSSION TITLE
Implementing HID for Braille Display to use with NVDA

### DISCUSSION THREAD

**Discussion by kumaaarrr** [nvaccess/nvda#14848] [General]
Hello, currently I'm working on a Braille Display project and I want to implement HID in my device. I'm very new to this USB and HID protocols and have no clue on how to start. Can someone direct me on how to proceed like what documentation should I refer or how should I implement it in order to make it work with NVDA.

If possible any previous implementation example on this would help me a lot. Thanks

**Comment by seanbudd** [comment]
A discussion for this was just opened - I suggest participating in this discussion instead: https://github.com/nvaccess/nvda/discussions/14829
